tasks/create-advisory/create-advisory.yaml

```diff
@@ -72,18 +70,73 @@ spec:
 
         pipelinerun_label="internal-services.appstudio.openshift.io/pipelinerun-uid"
 
+        # only 2 gitlab instances are permitted...prod and staging
+        # these are their secret names
+        #
+        prodSecretName="create-advisory-secret"
+        stagingSecretName="create-advisory-stage-secret"
+        #
+        # detect which one to use based on repositories specified
+        #
+        pending_repositories=$(jq -r '.spec.data.mapping.components[] | select(.repository | contains("quay.io/redhat-pending/")) | .repository' $(workspaces.data.path)/$(params.releasePlanAdmissionPath))
```

Comment:
I believe we should be checking the data file here, not the RPA

Reply:
fixed